docs/docs/forms.md

```diff
@@ -61,7 +62,7 @@ class Form extends React.Component {
 ReactDOM.render(<Form />, document.getElementById('root'));
 ```
 
-[Try it on CodePen.](https://codepen.io/ericnakagawa/pen/QKkJva?editors=0010)
+[Try it on Codepen](https://codepen.io/ericnakagawa/pen/QKkJva?editors=0010)
```

Comment:
I intentionally changed it to "CodePen" the last time because [this is how the service is capitalized](https://twitter.com/codepen):

<img width="270" alt="screen shot 2016-10-25 at 12 08 33" src="https://cloud.githubusercontent.com/assets/810438/19683796/c6334be6-9aab-11e6-900c-5dbfca2484a7.png">

What is the reason for changing it back?